backend/runner/mail/slow_query_weekly_mail.go

```diff
@@ -204,15 +195,50 @@ func (s *SlowQueryWeeklyMailSender) sendEmail(ctx context.Context, now time.Time
 			continue
 		}
 
-		users := utils.GetUsersByRoleInIAMPolicy(ctx, s.store, api.ProjectOwner, policyMessage.Policy)
+		s.sendEmailToUsersByPolicy(
+			ctx,
+			api.ProjectOwner,
+			policyMessage.Policy,
+			storeValue,
+			fmt.Sprintf("%s database slow query weekly report %s", project.Title, generateDateRange(now)),
+			body,
+		)
+	}
+}
+
+func (s *SlowQueryWeeklyMailSender) getEmailListByRoles(
+	ctx context.Context,
+	policy *storepb.IamPolicy,
+	roles ...api.Role,
+) []string {
+	usersMap := make(map[string]bool)
+	for _, role := range roles {
+		users := utils.GetUsersByRoleInIAMPolicy(ctx, s.store, role, policy)
 		for _, user := range users {
 			if user.Email == api.SystemBotEmail {
```

Comment:
user.Email != api.EndUser

So it ignores the system bot and service account. (It intends to send to human to take action anyway)

Reply:
Right